dace/sdfg/work_depth_analysis/work_depth.py

```diff
@@ -5,7 +5,7 @@
 import argparse
 from collections import deque
 from dace.sdfg import nodes as nd, propagation, InterstateEdge
-from dace import SDFG, SDFGState, dtypes
+from dace import SDFG, SDFGState, dtypes, int64
```

Comment:
`dtypes` is already imported, so using `dtypes.int64` instead is preferred if possible.